planner/core/integration_test.go

```diff
@@ -7554,6 +7554,26 @@ func TestEnableTiFlashReadForWriteStmt(t *testing.T) {
 	checkMpp(rs)
 }
 
+func TestPointGetWithSelectLock(t *testing.T) {
```

Comment:
Not just a test for a primary key, a test for a unique key is also necessary.

Reply:
updated